kudos.py

```diff
@@ -71,8 +71,10 @@ def do_comment(browser, csrf_token, activity_id, comment):
 		if len(avatars) > 0:
 			avatar = avatars[0]
 			profile_link = avatar['href']
+                        # Get the athlete from the url i.e '/athletes/1234567
+			athlete_id = profile_link[len('/athletes/'):]
```

Comment:
Here you could add `athlete_id = int(ahtlete_id)` to cast it to an it, so that the values can be int's in the config file.